docs/licensing.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: licensing
+append_help_link: true
+description: >-
+  Learn about licenses of r2c products.
+---
+
+# Licensing
+
+This document provides an overview of the licenses of important products created by r2c. See the list of the products and their licenses below:
+
+<dl>
+  <dt>Semgrep CLI</dt>
+    <dd>Semgrep CLI is an open source project licensed under <a href="https://github.com/returntocorp/semgrep/blob/develop/LICENSE">LGPL 2.1</a>.</dd>
+  <dt>Semgrep Supply Chain</dt>
+    <dd>Proprietary. See <a href="https://semgrep.dev/terms">Terms of Service</a>.</dd>
+  <dt>Semgrep App</dt>
+    <dd>Proprietary. See <a href="https://semgrep.dev/terms">Terms of Service</a>.</dd>
+  <dt>DeepSemgrep</dt>
```

Comment:
Since DeepSemgrep is an extension to the Semgrep engine, perhaps nest under that? As indented here I think it could be interpreted as a stand-alone product.

Reply:
I will do that, thank you!

Reply:
Now added as:

```html
  <dt>Semgrep CLI</dt>
    <dd>Semgrep CLI is an open source project licensed under <a href="https://github.com/returntocorp/semgrep/blob/develop/LICENSE">LGPL 2.1</a>. Proprietary part of Semgrep is DeepSemgrep, for more information see <a href="https://semgrep.dev/terms">Terms of Service</a>.</dd>
```